samples/app-anonymous-users/csharp/Controllers/AuthController.cs

```diff
@@ -0,0 +1,115 @@
+﻿// <copyright file="AuthController.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// </copyright>
+using AnonymousUsers.Helper;
+using AnonymousUsers.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace AnonymousUsers.Controllers
+{
+    public class AuthController : Controller
+    {
+        /// <summary>
+        /// Represents a set of key/value application configuration properties.
+        /// </summary>
+        public readonly IConfiguration _configuration;
+        private readonly IHttpClientFactory _httpClientFactory;
+        private readonly IHttpContextAccessor _httpContextAccessor;
+
+        public AuthController(
+            IConfiguration configuration,
+            IHttpClientFactory httpClientFactory,
+            IHttpContextAccessor httpContextAccessor)
+        {
+            _configuration = configuration;
+            _httpClientFactory = httpClientFactory;
+            _httpContextAccessor = httpContextAccessor;
+        }
+
+        /// <summary>
+        /// Get facebook profile of user.
+        /// </summary>
+        /// <param name="accessToken">Token</param>
+        /// <returns></returns>
+        [HttpPost]
+        [Route("getFbAccessToken")]
+        public async Task<JsonResult> GetFacebookAuthToken(string accessToken)
```

Comment:
Similar to my comment in the frontend, from my reading  this doesn't return a Facebook token, but instead returns a Facebook profile, can you rename this function/api path to match that?

Reply:
Updated your feedback.